frontend/src/common/PageWithTabs.tsx

```diff
@@ -185,17 +185,19 @@ interface TabPanelProps {
 
 function CustomTabPanel(props: TabPanelProps) {
   const { children, tabKey, currentTab, ...other } = props
-  const theme = useTheme()
 
   return (
     <div role='tabpanel' hidden={currentTab !== tabKey} {...other}>
       {currentTab === tabKey && (
         <Box
-          sx={{
-            backgroundColor: theme.palette.mode === 'light' ? 'white' : grey[800],
+          sx={(theme) => ({
+            backgroundColor: grey[800],
```

Comment:
Maybe add a todo to use `theme.applyStyles` to set this for dark mode only in a future PR?